### DISCUSSION TITLE
<title> Unable to open Codespace on GitHub

### DISCUSSION THREAD

**Discussion by sdjeverywherewhenever** [community/community#45690] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello, I've been trying to open my cloud-based github codespace for the course im taking rn but i've been stuck on this page. I've tried refreshing and using safari / chrome. nothing seems to work. This issue is really hindering the progress of my course.

Any comments or thoughts will be very much appreciated, thank you!

edit:
codespace opened with the following message: " This codespace is currently running in recovery mode due to a configuration error. Please review the creation logs, update your devcontainer configuration as needed, and run the "Rebuild Container" command to rectify."

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sdjeverywherewhenever ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by gdomingu** [comment]
```
This codespace is currently running in recovery mode due to a configuration error.
```
This error can happen when you have an issue in your devcontainer.json file. I would recommend commenting out certain lines of devcontainer.json to help you narrow down where the issue may be coming from. 

[Doc for dev containers.](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers)
